src/Orchard.Web/Modules/Orchard.Search/Controllers/SearchController.cs

```diff
@@ -45,23 +46,24 @@ public class SearchController : Controller {
         public ILogger Logger { get; set; }
         dynamic Shape { get; set; }
 
-        public ActionResult Index(PagerParameters pagerParameters, string q = "") {
+        public ActionResult Index(PagerParameters pagerParameters, string searchIndex = null, string q = "") {

```

Comment:
Breaking Change. I think if we are going to do this, we might want to think about creating a SearchParameters object?